exampleSite/content/fragments/search/search.md

```diff
@@ -4,7 +4,7 @@ weight = 110
 title = "Search"
 background = "secondary"
 
-#title = ""
-#subtitle = ""
+title = "Search Fragment"
+subtitle = "Searches through all content fragments"
```

Comment:
Both title and subtitle aren't used on the rendered page aka probably not used within the layout yet.